cmd/secret/cmd/root.go

```diff
@@ -0,0 +1,82 @@
+// Package cmd provides functions set and get with cobra library
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-itools-internship/go-secret/pkg/crypto"
+	"github.com/go-itools-internship/go-secret/pkg/io/storage"
+	"github.com/go-itools-internship/go-secret/pkg/provider"
+	"github.com/spf13/cobra"
+)
+
+type root struct {
+	key       *string
+	cipherKey *string
+	value     *string
+	path      *string
+	rootCmd   *cobra.Command
```

Comment:
It would increase readability if no duplication in naming occur. `cmd` will be okay for understanding